types/node/buffer.d.ts

```diff
@@ -1701,6 +1701,8 @@ declare module "buffer" {
              * @return A reference to `buf`.
              */
             fill(value: string | Uint8Array | number, offset?: number, end?: number, encoding?: BufferEncoding): this;
+            fill(value: string | Uint8Array | number, offset?: number, encoding?: BufferEncoding): this;
+            fill(value: string | Uint8Array | number, encoding?: BufferEncoding): this;
```

Comment:
In general, it's better not to type supplemental overloads variadically. Changing these signatures to have mandatory parameters still encodes the same behaviour, but reduces the burden of overload resolution in certain circumstances.

```suggestion
            fill(value: string | Uint8Array | number, offset: number, encoding: BufferEncoding): this;
            fill(value: string | Uint8Array | number, encoding: BufferEncoding): this;
```

Reply:
Thanks! I fixed in [98ec455](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/73143/commits/98ec455ecd3224a39867200df1309640cd7e1680).